catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v1_1_0/catalog/source/WfsFilterDelegateTest.java

```diff
@@ -0,0 +1,2245 @@
+
+    Writer writer = new StringWriter();
+    Marshaller marshaller = JAXB_CONTEXT.createMarshaller();
+    marshaller.marshal(getFilterTypeJaxbElement(filter), writer);
+    String xml = writer.toString();
+
+    assertXMLEqual(propertyGreaterThanXml, xml);
+  }
+
+  @Test
+  public void testPropertyIsGreaterThanLong() throws JAXBException, SAXException, IOException {
+    List<String> mockTextProps = new ArrayList<>();
+    mockTextProps.add(MOCK_PROPERTY);
+    when(featureMetacardType.getProperties()).thenReturn(mockTextProps);
+    when(featureMetacardType.getAttributeDescriptor(MOCK_PROPERTY))
+        .thenReturn(
+            new FeatureAttributeDescriptor(
+                MOCK_PROPERTY, MOCK_PROPERTY, true, true, true, true, BasicTypes.LONG_TYPE));
+    WfsFilterDelegate delegate =
+        new WfsFilterDelegate(featureMetacardType, SUPPORTED_GEO, SRS_NAME);
+    long literal = 1L;
+    FilterType filter = delegate.propertyIsGreaterThan(MOCK_PROPERTY, literal);
+    Writer writer = new StringWriter();
+    Marshaller marshaller = JAXB_CONTEXT.createMarshaller();
+    marshaller.marshal(getFilterTypeJaxbElement(filter), writer);
+
+    assertXMLEqual(propertyGreaterThanXml, writer.toString());
+  }
+
+  @Test
+  public void testPropertyIsGreaterThanFloat() throws JAXBException, SAXException, IOException {
+    List<String> mockTextProps = new ArrayList<>();
+    mockTextProps.add(MOCK_PROPERTY);
+    when(featureMetacardType.getProperties()).thenReturn(mockTextProps);
+    when(featureMetacardType.getAttributeDescriptor(MOCK_PROPERTY))
+        .thenReturn(
+            new FeatureAttributeDescriptor(
+                MOCK_PROPERTY, MOCK_PROPERTY, true, true, true, true, BasicTypes.LONG_TYPE));
+    WfsFilterDelegate delegate =
+        new WfsFilterDelegate(featureMetacardType, SUPPORTED_GEO, SRS_NAME);
+    float literal = 1.0F;
+    FilterType filter = delegate.propertyIsGreaterThan(MOCK_PROPERTY, literal);
+    Writer writer = new StringWriter();
+    Marshaller marshaller = JAXB_CONTEXT.createMarshaller();
+    marshaller.marshal(getFilterTypeJaxbElement(filter), writer);
+
+    assertXMLEqual(propertyGreaterThanXmlDecimal, writer.toString());
+  }
+
+  @Test
+  public void testPropertyIsGreaterThanDouble() throws JAXBException, SAXException, IOException {
+    List<String> mockTextProps = new ArrayList<>();
+    mockTextProps.add(MOCK_PROPERTY);
+    when(featureMetacardType.getProperties()).thenReturn(mockTextProps);
+    when(featureMetacardType.getAttributeDescriptor(MOCK_PROPERTY))
+        .thenReturn(
+            new FeatureAttributeDescriptor(
+                MOCK_PROPERTY, MOCK_PROPERTY, true, true, true, true, BasicTypes.LONG_TYPE));
+    WfsFilterDelegate delegate =
+        new WfsFilterDelegate(featureMetacardType, SUPPORTED_GEO, SRS_NAME);
+    double literal = 1.0;
+    FilterType filter = delegate.propertyIsGreaterThan(MOCK_PROPERTY, literal);
+    Writer writer = new StringWriter();
```

Comment:
the following 3 lines are repeated multiple times, i wonder if can be extracted out of the test methods